README.md

```
@@ -77,28 +77,84 @@ A aplicação segue uma arquitetura **MVC (Model-View-Controller)** com separaç
 4. **Execute localmente via Jetty:**
    ```bash
    mvn jetty:run
    ```
-   Acesse a aplicação em: `http://localhost:8080`
+
+5. **Acesse a aplicacao:**
+
+   Abra o navegador em `http://localhost:8080/auth/login`.
+
+   Se executou o `data.sql`, use as credenciais de teste:
+   | Campo | Valor |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)